packages/docs/src/components/DocsCode.vue

```diff
@@ -9,6 +9,20 @@ import 'prismjs/components/prism-bash'
 import VuePrismComponent from 'vue-prism-component'
 
 export default {
+  data() {
```

Comment:
Would be nice to have a comment on why exactly do you need to rerender VuePrismComponent on code prop change. Why not just pass code prop to VuePrismComponent?

Reply:
Yeah, doesn't seem appropriate.